.github/workflows/actions/entrypoint.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
+
+yum install -y udunits2-devel
+PYTHONS=("cp36-cp36m" "cp37-cp37m" "cp38-cp38" "cp39-cp39")
```

Comment:
@ocefpaf We've now dropped support for `cp36` 🎉

Reply:
Removed.